### DISCUSSION TITLE
How can I get my transaction id, When I push my tx to node(use push_tx method)?

### DISCUSSION THREAD

**Discussion by Geapefurit** [Chia-Network/chia-blockchain#18613] [Ideas]
or tall me how to cal the transaction id from spend_bundle?

**Comment by BrandtH22** [comment]
Hey @Geapefurit , the transaction id used in the mempool is a hash of the spend bundle streamed to bytes32. 

Let us know if you run into any issues with deriving the tx id

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
